docs/api.rst

```diff
@@ -454,24 +467,193 @@ in these models see :ref:`sec_api_mutation_matrix_models_details`.
+
+where :math:`M` is an overall scaling factor chosen so that the largest 
+row sum in the matrix :math:`B_{ij} \pi_j /M` is equal to one.
+In this model, :math:`M = 1.726203705809619`.
+This implementation as a :class:`.MatrixMutationModel` has transition matrix
+:math:`P_{ij} = B_{ij} \pi_{j} / M`, and :math:`P_{ii}` chosen so rows sum to one,
+and root distribution equal to the equilibrium frequencies, :math:`\pi`,
+so the matrix :math:`B` can be recovered as follows:
+
+.. code-block:: python
+
+   model = msprime.BLOSUM62()
+   M = 1.726203705809619
+   B = M * model.transition_matrix / model.root_distribution
+
+Citation: The values of :math:`B` and :math:`\pi` used here were copied from
+`Seq-Gen <http://tree.bio.ed.ac.uk/software/seqgen/>`_.
+The original citation is: *Henikoff, S., and J. G. Henikoff. 1992. PNAS USA
+89:10915-10919,* and the numerical values were provided in: *Yu,Y.-K.,
+Wootton,J.C. and Altschul,S.F. (2003) The compositional adjustment of amino
+acid substitution matrices. Proc. Natl Acad. Sci., USA, 100, 15688–15693.*
+
 .. autoclass:: msprime.PAMMutationModel()
 
-.. autoclass:: msprime.MatrixMutationModel()
+The model is parameterized by a 20-by-20 symmetric matrix of relative rates,
+:math:`B`, and a vector of amino acid equilibrium frequencies, :math:`\pi`
+(for the precise order, see this model's :attr:`.alleles` attribute).
+If this mutation model is used with a ``mutation_rate`` of :math:`\mu`, then
+the mutation rate from the i-th to the j-th allele is :math:`Q_{ij}`, where
 
+:math:`Q_{ij} = \frac{\mu}{M} B_{ij} \pi_j,`
 
-++++++++++++++++++++++
-Other Mutation Models
-++++++++++++++++++++++
+where :math:`M` is an overall scaling factor chosen so that the largest 
+row sum in the matrix :math:`B_{ij} \pi_j /M` is equal to one.
+In this model, :math:`M = 1.78314862248`.
+This implementation as a :class:`.MatrixMutationModel` has transition matrix
+:math:`P_{ij} = B_{ij} \pi_{j} / M`, and :math:`P_{ii}` chosen so rows sum to one,
+and root distribution equal to the equilibrium frequencies, :math:`\pi`,
+so the matrix :math:`B` can be recovered as follows:
+
+.. code-block:: python
+
+   model = msprime.PAM()
+   M = 1.78314862248
+   B = M * model.transition_matrix / model.root_distribution
+
+Citation: The values of :math:`B` and :math:`\pi` used here were copied from
+`Seq-Gen <http://tree.bio.ed.ac.uk/software/seqgen/>`_,
+and follow the "Dayhoff DCMut" model as described in *Kosiol, C., and Goldman,
+N. 2005. Different versions of the Dayhoff rate matrix. Molecular Biology and
+Evolution 22:193-199.* The original citation is: *Dayhoff, M.O., Schwartz,
+R.M., Orcutt, B.C. (1978). A model of evolutionary change in proteins. Atlas of
+Protein Sequence Structures, Vol. 5, Suppl. 3, National Biomedical Research
+Foundation, Washington DC, pp. 345-352.*
 
-.. autoclass:: msprime.InfiniteAllelesMutationModel()
 
-.. autoclass:: msprime.SLiMMutationModel()
 
 .. _sec_api_mutation_matrix_models_details:
```

Comment:
I think this is my bad, but this sections title should be "Matrix Mutation Models Details" not  "Mutation Matrix Models Details"